Zend/zend_execute.h

```diff
@@ -342,11 +345,13 @@ static zend_always_inline zend_execute_data *zend_vm_stack_push_call_frame_ex(ui
 
 	if (UNEXPECTED(used_stack > (size_t)(((char*)EG(vm_stack_end)) - (char*)call))) {
 		call = (zend_execute_data*)zend_vm_stack_extend(used_stack);
+		ZEND_UNPOISON_MEMORY_REGION(call, used_stack);
```

Comment:
Oh, my only comment didn't go through. ^^ I was wondering if it may make sense to move this to `zend_vm_init_call_frame()`, which would avoid a chunk of the `ZEND_UNPOISON_MEMORY_REGION()` calls.